app/files.go

```diff
@@ -87,6 +102,20 @@ func main() {
 
 	e.Stop()
 
+    if *memprofile != "" {
+        f, err := os.Create(*memprofile)
+		if err != nil {
+			fmt.Println(err.Error())
```

Comment:
Rather than just outputting the error from os.Create, perhaps something more descriptive like: 'Failed to create CPU profiling output file .....' or something like that?